Drive/rover_drive_system.hpp

```diff
@@ -120,6 +134,8 @@ class RoverDriveSystem
   /// @return true if rover is able to retrieve data from all the wheels
   bool GetRoverData()
```

Comment:
Maybe call this `PrintRoverData()` because its not retrieving data from the motors or the rover. Its just printing it out. Which you won't see if you are operating the rover from afar without a connection to UART.

Reply:
done - also changed it to a void function